packages/loader/container-loader/src/debugLogger.ts

```diff
@@ -0,0 +1,113 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	ITelemetryBaseEvent,
+	ITelemetryBaseLogger,
+	ITelemetryProperties,
+} from "@fluidframework/core-interfaces";
+import { performance } from "@fluidframework/common-utils";
+import { debug as registerDebug, IDebugger } from "debug";
+import {
+	ITelemetryLoggerExt,
+	ITelemetryLoggerPropertyBags,
+	createMultiSinkLogger,
+	eventNamespaceSeparator,
+	formatTick,
+} from "@fluidframework/telemetry-utils";
+
+/**
+ * Implementation of debug logger
+ */
+export class DebugLogger implements ITelemetryBaseLogger {
```

Comment:
i've actually heard rumblings from some partners that they don't like this being here by default, as they need to pay the bundle cost for it, but don't use it, as it doesn't log their properties. I think making it internal to this package, and ensuring it is only used here would make removing it easier if we think that is a path we'd like to take. We could also move this to its own package if we think we'd still want to provide it. in that case we'd probably remove the mixin stuff, and just provide it as a base logger.